examples/.gitignore

```diff
@@ -1,2 +1,4 @@
 # Ignore the bazel symlinks
 /bazel-*
+# ignore protoc generated files
+*.pb.go
```

Comment:
why this need to be ignored?

Reply:
Since it's auto-generated from the  .proto file. This is only as far as the /examples directory is concerned.